src/frontends/tensorflow/src/op/is_inf.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "op_table.hpp"
+#include "openvino/opsets/opset10.hpp"
+
+using namespace std;
+using namespace ov::opset10;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+
+OutputVector translate_is_inf_op(const NodeContext& node) {
```

Comment:
why do we need a separate translator for this? can we use translate_unary_op function in op_table.cpp?

Reply:
Let us do this

Reply:
Done